README.md

```diff
@@ -1 +1,22 @@
 # AllegroPlanet
+
+Setup guide for developers
+----
+
+Here is a list of prerequisites 
+for developers on Debian-like systems:
+
+sudo apt install \
+	sqlite3 \
+	ruby \
+	ruby-dev \
+	nodejs \
+	postgresql \
+	libpq-dev
```

Comment:
I would just put these in a single line and format the terminal commands with ``` so it looks good in Markdown:

```
sudo apt install sqlite3 ruby ruby-dev nodejs postgresql libpq-dev
gem install rails
```

etc.